geotrek/trekking/tests/test_parsers.py

```diff
@@ -110,6 +117,9 @@ def test_multilinestring_with_hole(self):
     b'356612.3778 6689511.5083, 356447.7689 6689502.3634)'
 )
 
+# return # Produce a small red dot
+IMG_FILE = b'iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg=='
```

Comment:
You could also import this variable from `geotrek/common/utils/testdata`

Reply:
Yes, I guess I did not want to add another dependency on the `common` module. But hey, It's what the `common` module is for!
I updated my code to import the test data from `common`.